const p = new Promise((res, reject) => {
    reject(Error('Ошибка!'))
})